### ISSUE TITLE
Settings for Train

### ISSUE THREAD

**Issue by SWRDFK** [ChengpengChen/temp#1]
Author, thank you for your work. I got the error that " No such file or directory: 'Reid-mask-master/log/market/baseline-16x4-softmax-triplet-erase-0817-v2.log'". So how can I modify the config.yml (as following) for train.py? 

    #settings for model
    prefix: 'baseline-16x4-softmax-triplet-erase-0817-v2'

Waiting for your reply.

**Comment by ChengpengChen** [comment]
> Author, thank you for your work. I got the error that " No such file or directory: 'Reid-mask-master/log/market/baseline-16x4-softmax-triplet-erase-0817-v2.log'". So how can I modify the config.yml (as following) for train.py?
> 
> ```
> #settings for model
> prefix: 'baseline-16x4-softmax-triplet-erase-0817-v2'
> ```
> Waiting for your reply.

You may have to make dirs like 'log/market' and 'models/market' manually. Sorry for the inconvenience.

**Comment by SWRDFK** [comment]
Sorry, I don't have this file like "baseline-16x4-softmax-triplet-erase-0817-v2" or "pretrain_models/resnet-50". Can you provide me with it or how can I make it?

    #settings for model
    prefix: 'baseline-16x4-softmax-triplet-erase-0817-v2'

    network: "resnet_s16"

    model_load_prefix: "pretrain_models/resnet-50"  #"models/duke/baseline-s16-erase-2loss-1.0"
    model_load_epoch: 0
    #model_load_epoch: 126
    #model_load_prefix: "inception-bn"

**Comment by ChengpengChen** [comment]
> Sorry, I don't have this file like "baseline-16x4-softmax-triplet-erase-0817-v2" or "pretrain_models/resnet-50". Can you provide me with it or how can I make it?
> 
> ```
> #settings for model
> prefix: 'baseline-16x4-softmax-triplet-erase-0817-v2'
> 
> network: "resnet_s16"
> 
> model_load_prefix: "pretrain_models/resnet-50"  #"models/duke/baseline-s16-erase-2loss-1.0"
> model_load_epoch: 0
> #model_load_epoch: 126
> #model_load_prefix: "inception-bn"
> ```

The prefix like "baseline-16x4-softmax-triplet-erase-0817-v2" is just for indicating the specific running of the code, it is used as the name of log file. We don't have to provide it.

As for the pretrained models, you can download it [here](https://pan.baidu.com/s/115Q4T9zduhdCxXI8ONXrvQ). Hope it helps :)
